Packs/Coralogix/README.md

```diff
@@ -0,0 +1,130 @@
+## Overview
```

Comment:
This looks like the correct one, but it should be located under:
Packs/Coralogix/Integrations/Coralogix/README.md

Reply:
Moved. Tnx (: